shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/BridgeIngressServiceImpl.java

```diff
@@ -84,7 +84,7 @@ public Deployment fetchOrCreateBridgeIngressDeployment(BridgeIngress bridgeIngre
         environmentVariables.add(new EnvVarBuilder().withName(KafkaConfigurationCostants.KAFKA_CLIENT_ID_ENV_VAR).withValue(kafkaConfigurationProvider.getClient()).build());
         environmentVariables.add(new EnvVarBuilder().withName(KafkaConfigurationCostants.KAFKA_CLIENT_SECRET_ENV_VAR).withValue(kafkaConfigurationProvider.getSecret()).build());
         environmentVariables.add(new EnvVarBuilder().withName(KafkaConfigurationCostants.KAFKA_SECURITY_PROTOCOL_ENV_VAR).withValue(kafkaConfigurationProvider.getSecurityProtocol()).build());
-        environmentVariables.add(new EnvVarBuilder().withName(Constants.BRIDGE_INGRESS_BRIDGE_ID_CONFIG_ENV_VAR).withValue(bridgeIngress.getSpec().getId()).build());
+        environmentVariables.add(new EnvVarBuilder().withName(KafkaConfigurationCostants.KAFKA_TOPIC_ENV_VAR).withValue(InternalKafkaTopicNameBuilder.build(bridgeIngress.getSpec().getId())).build());
```

Comment:
@r00ta I know this is a draft, but I just want to make sure that we're aligned on the topic/cluster/service account to use being exposed via the DTO sent from the Manager --> Shard?

Reply:
Hi @robpblake , I talked with @kostola about this yesterday. Since the topic is on our internal cluster and is under our control we can establish a convention on the topic name to use instead of passing the topic name around. This convention is implemented in that `InternalKafkaTopicNameBuilder`. 
But, if you think we should implement include the topic name in the DTO and pass it around let me know and I'll change accordingly. 
Note that I created this MGDOBR-247 and I'm not using MGDOBR-170/171/172

Reply:
@r00ta Yes I think the Manager should provide the details via the API it exposes to the Shard of how the Shard should connect the Ingress and Processors to the internal event bus Kafka Topic for the OpenBridge Instance. 

I would expect the Manager to expose via the API for each OpenBridge instance:

- topic to use
- client id and secret to use to connect
- security protocols to use to connect
- broker urls to use to connect

That means we can completely change our implementation of how we fulfil the Kafka dependency within the Fleet Manager and the Shard doesn't need to change at all.

Whilst I appreciate most of this information will be hardcoded in the Manager at the moment (apart from the topic name), over-time this will change.

Reply:
Ok @robpblake, if this is the design you want me to implement, I'll do it.